Ide.slnf

```diff
@@ -39,7 +39,6 @@
       "src\\Compilers\\Core\\AnalyzerDriver\\AnalyzerDriver.shproj",
       "src\\Compilers\\Core\\CodeAnalysisTest\\Microsoft.CodeAnalysis.UnitTests.csproj",
       "src\\Compilers\\Core\\MSBuildTaskTests\\Microsoft.Build.Tasks.CodeAnalysis.UnitTests.csproj",
-      "src\\Compilers\\Core\\MSBuildTask\\Microsoft.Build.Tasks.CodeAnalysis.csproj",
```

Comment:
The removed projects just didn't exist in the solution any more. They were lingering in the slnf and only finally removed when I hit "update solution filter" in VS.

Reply:
Will send a separate PR to clean this up.